scripts/levels.js

```diff
@@ -1689,97 +1692,7 @@ function traceRay()
 	}
 } 
 
-function traceRay0()                // for tracing the ray path
```

Comment:
Why have you removed this function?

Reply:
Sorry that was accidental, I thought that TraceRay0() was a copy of TraceRay().

Reply:
Can you reject this pull request so I can submit a new one?  
I re-added the deleted code

Reply:
Changes made to a branch used for a pull request will reflect on the pull request until the pull request is successfully merged.